runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java

```diff
@@ -52,9 +201,23 @@ public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plug
 	{
 		panelComponent.getChildren().add(TitleComponent.builder().text("Coordinate Clue").build());
 
-		panelComponent.getChildren().add(LineComponent.builder()
-			.left("Click the clue scroll along the edge of your world map to see where you should dig.")
-			.build());
+		String solution = CLUES.get(location);
+
+		if (solution != null)
+		{
+			panelComponent.getChildren().add(LineComponent.builder()
+				.left(solution)
+				.build());
+
+			panelComponent.getChildren().add(LineComponent.builder().left("").build());
+			panelComponent.getChildren().add(TitleComponent.builder().text("Marked on world map.").build());
```

Comment:
You should use same message in both cases and in case there is solution just prepend those 2 lines above to it.

Reply:
I'm not sure to understand.
But if we don't know the coordinate, show the default message.
And if we know the coordinate show the default message + the solution ? 
![CoordinateClue](https://user-images.githubusercontent.com/45672110/54274307-0dbefd00-4588-11e9-969b-002ce2009aec.png)

Reply:
Yeah exactly. Maybe alter default message to be shorter

Reply:
I think the actual message is fine, I make sure to not write to much in the solution.
I did a master clue scroll and got the jogre dig spot but the solution was so long that it's impossible to read all of it on fixed mode so i try to make the solution smaller than that one.

But still I think the number of plugins on and at the same spot as the clue helper can make it go out of screen.

Anyway here with smaller default solution (3 lines instead of 5).
![CoordinateClue](https://user-images.githubusercontent.com/45672110/54278164-f127c280-4591-11e9-8dc5-ba707f46f72a.png)

Choose the one you prefer and I'll update the PR.

Thanks

Reply:
I would put the detailed description at start, otherwise it looks fine.

Reply:
Any update on this @WoneTooPhree ?

Reply:
Sorry here are the changes.